pkg/codegen/go/gen.go

```diff
@@ -241,8 +244,18 @@ func (pkg *pkgContext) tokenToType(tok string) string {
 		contract.AssertNoErrorf(err, "Unable to determine package root")
 	}
 
+	var headerImportPath string
+	if pkg.externalPackageInfo != nil {
+		pkgInfo := pkg.externalPackageInfo[components[0]]
```

Comment:
Is the package guaranteed to be `pkg.externalPackageInfo`? usually it's better to check if the element is there
```go
if pkgInfo, ok := pkg.externalPackageInfo[components[0]]; ok {
  headerImportPath = path.Join(pkgInfo.ImportBasePath, mod)
}
```